app/controllers/admin/configs_controller.rb

```diff
@@ -129,10 +129,26 @@ class ConfigsController < Admin::ApplicationController
         home_feed_minimum_score
       ].freeze
 
+    IMAGE_FIELDS =
+      %w[
+        main_social_image
+        logo_png
+        secondary_logo_url
+        campaign_sidebar_image
+        mascot_image_url
+        mascot_footer_image_url
+        onboarding_logo_image
+        onboarding_background_image
+        onboarding_taskcard_image
+      ].freeze
+
+    VALID_URL = %r{\A(http|https)://([/|.|\w|\s|-])*.(?:jpg|gif|png)}.freeze
```

Comment:
@juliannatetreault quick one, do we not accept jpeg?

Reply:
@Ridhwana I'm going to remove the part of the validation that checks explicitly for the image URL and leave the validation to just validate the URL per yours and @rhymes' suggestion, so this won't matter in a couple of minutes. 😊